tightbind/CMakeLists.txt

```diff
@@ -86,6 +83,15 @@ endif(NOT MSVC)
 
 # Add the executable and link to the C math library
 add_executable(bind main.c)
+
+option(BIND_NAME "Name of bind binary" OFF)
```

Comment:
for some reason setting the default (OFF) to something else doesn't work, hence ugly few lines below

Reply:
I think this whole block would be cleaner as:
```
if(NOT BIND_EXE_NAME)
  set(BIND_EXE_NAME bind)
  message("-- Writing bind binary to \"${BIND_EXE_NAME}\"")
  message("-- Use -DBIND_EXE_NAME=<name> to modify")
else()
  message("-- Writing bind binary to \"${BIND_EXE_NAME}\"")
endif()

set_target_properties(bind PROPERTIES OUTPUT_NAME ${BIND_EXE_NAME})
```